torchrl/data/tensordict/metatensor.py

```diff
@@ -70,7 +70,8 @@ def __init__(
         _is_shared: Optional[bool] = None,
         _is_memmap: Optional[bool] = None,
     ):
-
+        _is_tensordict = False
+        _repr = None
```

Comment:
If we want `repr` to be non-None only for tensordict, we may rename this variable to be something else?

Reply:
refactored! Thanks for the suggestion